NuRadioReco/utilities/noise.py

```diff
@@ -102,10 +102,7 @@ def rolled_sum_slicing(traces, rolling):
     sumtr = traces[0].copy()
     for i in range(1, len(traces)):
         r = rolling[i]
-        if r > 0:
-            sumtr[:-r] += traces[i][r:]
-            sumtr[-r:] += traces[i][:r]
-        elif r < 0:
+        if r != 0:
```

Comment:
Thanks for spotting and correcting. That does the correct thing now... 

A simple test would have avoided this in the first place :/
I verified it works for rolling<trace length...that should not happen, but one could put
`r = rolling[i]%len(sumtr)` for completeness (with that, `r` would never have been negative in the first place)

I used this to test:
```
def test_rolling_functions(n_tests=100, n_traces=5):
    """ tests for equality of output of `rolled_sum_roll` and `rolled_sum_slicing` """
    traces = [np.random.randint(-10,11,100) for i in range(n_traces)]

    for i in range(n_tests):
        rollings = np.random.randint(-100,100,n_traces)
        rollings -= rollings[0]

        assert np.alltrue(rolled_sum_roll(traces, rollings) == rolled_sum_slicing(traces, rollings)), "not equal"
```

Reply:
Thanks for the double check!